drawing.js

```diff
@@ -204,34 +207,13 @@ class Canvas {
     //draw energy level bars above nematodes
     if(world.energyBarOn) {
       world.forEachNematode(n => this.DrawEnergyLevel(n));
-
     }
 
-    // draw the selected nematode's neural network and stats
     if (world.selectedNematode != null){
```

Comment:
The comments are removed because that's not what the code does anymore. It's just easier and cleaner if code is delegated to the right spots instead of lumped all together in this drawing.js loop

Reply:
I just meant that there should be comments in the new code